### ISSUE TITLE
Shows an error window when starting.

### ISSUE THREAD

**Issue by alejx** [lhc70000/iina#119]
![7034680e-1779-4adf-a748-c8714e22710a](https://cloud.githubusercontent.com/assets/20237141/21765769/d53bb840-d6a5-11e6-8415-bc9b71238306.png)

The above error window will show up when opening IINA by chance. (Maybe about 50%)

**Comment by lhc70000** [comment]
Checked mpv's source code and still cannot tell why it failed. Just tried something to solve this problem.

**Comment by LeoNatan** [comment]
I am also seeing a similar error:

> Fatal error: MPV API error: "error setting option", Return value: -7. 
> The application will exit now.

Strange, as I already built and ran the app. I switch to #90 branch and I can no longer run the app.

**Comment by LeoNatan** [comment]
```
chkErr(mpv_set_option_string(mpv, MPVOption.Input.inputConf, inputConfPath))
```

Here, `inputConfPath` is `nil`.

**Comment by lhc70000** [comment]
Please leave a reply if you encountered this problem in v0.0.2.

**Comment by lhc70000** [comment]
Close due to fixed in [0.0.3](https://github.com/lhc70000/iina/releases/tag/v0.0.3).
